R/find_exports.R

```diff
@@ -2,18 +2,10 @@ find_exports <- function(clean_lns) {
   ids <- find_extendr_attrs_ids(clean_lns)
   start <- ids
   end <- dplyr::lead(ids, default = length(clean_lns) + 1L) - 1L
-  purrr::map2_dfr(
-    start,
-    end,
-    ~ extract_meta(clean_lns[seq(.x, .y, by = 1L)])
-  ) %>%
-    # Keeps only name, type (fn|impl) and lifetime of impl
-    # if present.
-    dplyr::transmute(
-      .data$name,
-      type = dplyr::if_else(is.na(.data$impl), "fn", "impl"),
-      .data$lifetime
-    )
+
+  purrr::map2_dfr(start, end, ~ extract_meta(clean_lns[.x:.y])) %>%
+    dplyr::mutate(type = dplyr::coalesce(.data$impl, .data$fn)) %>%
+    dplyr::select(dplyr::all_of(c("name", "type", "lifetime")))
```

Comment:
`.data$` right here is deprecated. Introduced warnings into testthat suite but silenced checks. This approach solves checks and tests.